### ISSUE TITLE
Cross-encoder distillation

### ISSUE THREAD

**Issue by raphaelsty** [lightonai/pylate#2]
Working on cross-encoder distillation

We want to distill cross-encoder into ColBERT,

- Update the training so we can distill logits pre-computed on option.
- Create a function which computes logits using a pre-trained cross-encoder in order to create a dataset with logits that can be used to distill soft labels to colbert.
